site/docs/latest/getting-started/concepts.md

```diff
@@ -76,7 +76,7 @@ The NameNode, however, served only as initial inspiration for BookKeeper. The ap
 
 ## Metadata storage
 
-BookKeeper requires a metadata storage service to store information related to [ledgers](#ledgers) and available bookies. BookKeeper currently uses [ZooKeeper](https://zookeeper.apache.org) for this and other tasks.
+The BookKeeper metadata store maintains all the metadata of the BookKeeper cluster, such as [ledger](#ledgers) metadata, available bookies, and so on. Currently, BookKeeper uses [ZooKeeper](https://zookeeper.apache.org) for  metadata storage. You can use one Zookeeper cluster for both BookKeeper metadata store and [Pulsar metadata store](https://pulsar.apache.org/docs/en/next/concepts-architecture-overview/#metadata-store). If you want to deploy Pulsar brokers connected to an existing BookKeeper cluster, you need to deploy separate ZooKeeper clusters for Pulsar metadata store and BookKeeper metadata store respectively.
```

Comment:
I am not sure we need to add here in BK website this reference to Pulsar.

Can you please explain why do we need this correction?
Can't we write it in Pulsar website?

Reply:
@eolivelli thanks for your comments.

> Can you please explain why do we need this correction?

Metadata store can be separated into the Pulsar metadata store and the BookKeeper metadata store?

For the Pulsar metadata store, I've added descriptions [here](https://github.com/apache/pulsar/pull/9422/files#diff-4619ea7c265622aba3a3a8356003ee23a23adf0942829badd35c4ca9183d3ab0L46).

> Can't we write it in Pulsar website?

I think the content of this PR is related to BookKeeper and its metadata store, so I add them here.

Reply:
I think we should remove the reference for Pulsar from BookKeeper since Pulsar depends on BK so reference should come from Pulsar doc to BK.

Reply:
@hsaputra thanks for your comments.

I've made the following changes:

- Remove this sentence "If you want to deploy Pulsar brokers connected to an existing BookKeeper cluster, you need to deploy separate ZooKeeper clusters for Pulsar metadata store and BookKeeper metadata store respectively." to [Pulsar doc](https://github.com/apache/pulsar/pull/9422/files#diff-4619ea7c265622aba3a3a8356003ee23a23adf0942829badd35c4ca9183d3ab0R48) 
- Keep the sentence "You can use one ZooKeeper cluster for both BookKeeper metadata store and [Pulsar metadata store](https://pulsar.apache.org/docs/en/next/concepts-architecture-overview/#metadata-store). " here since it indicates users metadata store can be separated into BookKeeper metadata store and Pulsar metadata store and gives users a link to Pulsar if they want to explore more.

PTAL, thanks.

Reply:
Thanks for the response, @Anonymitaet . My initial comment was about mentioning Pulsar's metadata store here.

Apache BookKeeper is used in Apache Pulsar for the storage layer, but BookKeeper does not depends on Pulsar to run. Hence I question the needs to mentions Pulsar's metadata store in BK doc.

Reply:
@hsaputra thanks for your comments. I've removed the Pulsar metadata store contents from here, PTAL.